modules/plugin/coverage-multidim/netcdf/src/test/java/org/geotools/coverage/io/netcdf/NetCDFCRSTest.java

```diff
@@ -419,6 +446,43 @@ public void testUnsupportedGridMappingName() throws IOException {
         reader.dispose();
     }
 
+    @Test
+    public void testPreserveKM() throws Exception {
+        setFinalStaticField("CONVERT_AXIS_KM", false);
```

Comment:
Use CONVERT_AXIS_KM_KEY

Reply:
Nope. Here I'm modifying the static field through reflection since the execution order of tests affect the outcome. the Key is the property name to be passed as JAVA_OPT